client/my-sites/stats/stats-module/index.jsx

```diff
@@ -92,7 +92,8 @@ class StatsModule extends Component {
 				'/' +
 				siteSlug +
 				'?startDate=' +
-				period.startOf.format( 'YYYY-MM-DD' )
+				period.startOf.format( 'YYYY-MM-DD' ) +
+				'&summarize=1'
```

Comment:
The `View all` links always respect the selected period to navigate to the summary for **_one unit_**, which might not need the `summarize=1`. Do we plan to keep the behavior? For example, even if we select the date range from `Last Year` with the period chosen `week`, the `View all` link is still `/stats/week/posts/{site-slug}?startDate=2024-10-28`.

Reply:
Yes, I think you are right. The reason I'm adding summarize here is that, the passed `startDate` will be replaced by the date range as well I imagine. Otherwise the data on the card wouldn't match it on the summary page. However I think it could be done later.

Reply:
I reverted the change for now.